client/cache/error_code.go

```diff
@@ -123,6 +123,10 @@ var InvalidSsdbCacheValue = berror.DefineCode(4002022, moduleName, "InvalidSsdbC
 SSDB cache only accept string value. Please check your input.
 `)
 
+var InvalidLoadFunc = berror.DefineCode(4002023, moduleName, "InvalidLoadFunc", `
+Invalid loadFunc.
```

Comment:
Invalid load function for read-through pattern decorator.
You should pass a valid(non-nil) load function when initiate the decorator instance